app/models/room.rb

```diff
@@ -1,4 +1,23 @@
 class Room < ApplicationRecord
   has_many :messages, dependent: :destroy
   has_many :entries, dependent: :destroy
-end
+  def create_notification_message!(current_user, message_id)
+    # 自分以外にコメントしている人をすべて取得し、全員に通知を送る
+    temp_ids = Message.select(:user_id).where(room_id: id).where.not(user_id: current_user.id).distinct
```

Comment:
ここはおそらくEntryが正しそうです！
```suggestion
    temp_ids = Entry.select(:user_id).where(room_id: id).where.not(user_id: current_user.id).distinct
```